chart/README.md

```diff
@@ -541,14 +541,63 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on the resource demand such as CPU and Memory utilization.
```

Comment:
"up/down based on the resource demand such as CPU and Memory utilization." 
-> 
"up/down based on cpu and or memory utilization. "

Reply:
commit: c64cf6a87103364de113a7a8886112149f4b2556